moveit_core/exceptions/CMakeLists.txt

```diff
@@ -1,11 +1,10 @@
 set(MOVEIT_LIB_NAME moveit_exceptions)
 
 add_library(${MOVEIT_LIB_NAME} src/exceptions.cpp)
-set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
```

Comment:
please revert this change

Reply:
https://github.com/ros-planning/moveit2/pull/7/commits/06c07e054e551f759a81dbbe2c430be6c20a0100